recipes/coiled-runtime/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "coiled-runtime" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - python >=3.8
+    - pip
+    - coiled
+    - nodejs ==17.8.0
+    - nb_conda_kernels ==2.3.1
+    - numpy ==1.21.5
+    - pandas ==1.4.1
+    - dask ==2022.1.0
+    - distributed ==2022.1.0
+    - fsspec ==2022.3.0
+    - s3fs ==2022.3.0
+    - gcsfs ==2022.3.0
+    - pyarrow ==7.0.0
+    - python-snappy ==0.6.0
+    - jupyterlab ==3.3.2
+    - dask-labextension ==5.2.0
+    - lz4 ==4.0.0
+    - ipywidgets ==7.7.0
+    - numba ==0.55.1
+    - scikit-learn ==1.0.2
+    - python-graphviz ==0.19.1
+    - click <=8.0.4
+
+test:
+  imports:
+    - dask
+    - distributed
+    - pyarrow
+
+about:
+  home: https://github.com/coiled/coiled-runtime
+  summary:  "Simple and fast way to get started with Dask"
```

```suggestion
  summary:  "Simple and fast way to get started with Dask"
  description: |
      coiled-runtime is a metapackage that pins dask, and 
      other packages commonly used with dask, to a specific version combination that has been verified 
      to work under common use cases / stress testing
```

I think it's worth capturing the more detailed description @jrbourbeau gave.